### PR TITLE
Docs & CLI changes

### PR DESCRIPTION
**Pull request by m1-dev**
- Added missing `SlashCommandBuilder` imports for the guide's autocomplete command examples.
- The CLI will now panic to the console if multiple CommandKit configuration files exist.

### CODE REVIEW

File: packages/commandkit/bin/common.mjs [hunk 1]
```diff
@@ -72,16 +72,24 @@ const possibleFileNames = [
 export async function findCommandKitConfig(src) {
     const cwd = process.cwd();
     const locations = src ? [join(cwd, src)] : possibleFileNames.map((name) => join(cwd, name));
+    const foundConfigFiles = [];
 
     for (const location of locations) {
         try {
-            return await loadConfigInner(location);
+            const config = await loadConfigInner(location);
+            foundConfigFiles.push(location);
```
Comment (twlite) [inline packages/commandkit/bin/common.mjs:80]: why are we pushing config here? can't we just count the number?

File: packages/commandkit/bin/common.mjs [hunk 2]
```diff
@@ -72,16 +72,24 @@ const possibleFileNames = [
...
     }
 
-    panic('Could not locate commandkit config.');
+    if (foundConfigFiles.length > 1) {
+        panic(`Multiple configuration files found: ${foundConfigFiles.join(', ')}`);
+    } else if (foundConfigFiles.length === 0) {
+        panic('Could not locate CommandKit configuration file!');
+    }
+
+    return loadConfigInner(foundConfigFiles[0]);
```
Comment (twlite) [inline packages/commandkit/bin/common.mjs:92]: and why is this necessary here when we have already loaded the config above?

**Review by twlite — CHANGES_REQUESTED** [review]

File: packages/commandkit/bin/common.mjs (hunk 1 above)
Comment (twlite) [reply packages/commandkit/bin/common.mjs:80]: also I think this slows down the performance of bootstrapping process, do we really need this?

Comment (m1-dev) [reply packages/commandkit/bin/common.mjs:80]: > why are we pushing config here? can't we just count the number?

It's for actually logging the location of the file

Comment (m1-dev) [reply packages/commandkit/bin/common.mjs:80]: > also I think this slows down the performance of bootstrapping process, do we really need this?

It won't be that slow, it's for a rarer case anyway

File: packages/commandkit/bin/common.mjs (hunk 2 above)
Comment (m1-dev) [reply packages/commandkit/bin/common.mjs:92]: > and why is this necessary here when we have already loaded the config above?

I probably made a mistake, it shouldn't be loaded above

File: packages/commandkit/bin/common.mjs (hunk 1 above)
Comment (twlite) [reply packages/commandkit/bin/common.mjs:79]: > > why are we pushing config here? can't we just count the number?
> 
> It's for actually logging the location of the file

I dont think that's logging the location, `loadConfigInner` loads config data

Comment (m1-dev) [reply packages/commandkit/bin/common.mjs:79]: > I dont think that's logging the location, `loadConfigInner` loads config data

It does log the location
image

File: packages/commandkit/bin/common.mjs [hunk 3]
```diff
@@ -72,16 +72,23 @@ const possibleFileNames = [
 export async function findCommandKitConfig(src) {
     const cwd = process.cwd();
     const locations = src ? [join(cwd, src)] : possibleFileNames.map((name) => join(cwd, name));
+    const foundConfigFiles = [];
 
     for (const location of locations) {
         try {
-            return await loadConfigInner(location);
+            foundConfigFiles.push(location);
```
Comment (twlite) [inline packages/commandkit/bin/common.mjs:79]: this is completely wrong, you are just creating new array that exactly equals `locations` array.

**Review by twlite — CHANGES_REQUESTED** [review]

**Comment by m1-dev** [comment]
Closed since this isn't even needed
